src/mixins/buttons.css

```diff
@@ -0,0 +1,16 @@
+@import './focusable.css';
+button {
```

Comment:
This is copied from `uswds` with slight changes.

https://github.com/uswds/uswds/blob/c4c9410d54b537c6112037a36ed289d8b7e8940a/src/stylesheets/elements/_buttons.scss#L37-L51